registry/Dockerfile

```diff
@@ -6,7 +6,7 @@ ADD ./ /codebase
 WORKDIR /codebase
 
 RUN npm install --no-package-lock --no-progress --ignore-scripts
-RUN npm install sqlite3
+RUN npm install sqlite3 mysql
```

Comment:
is sqlite needed only for local development? if it's correct, we don't need to install it here, do we?